common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -79,6 +81,17 @@ public void onTitle(TitleSetTextEvent event) {
         }
     }
 
+    // One challenge is Nexus of Light does not display the scoreboard upon challenge completion
```

Comment:
Is the first "challenge" here just a remark about difficulties implementing the mod? In any other situation I would not have cared, but since this is about "challenges" in the game, it sounds a bit confusing. Perhaps replace with "difficulty"?

Reply:
It's referring to one of the challenge rooms in the raid

Reply:
Oh I see the confusion now, it should be "One challenge in" not "is"

Reply:
Ah, that makes sense. Somehow my brain failed to find that alternative interpretation.